apps/dsb-message-broker/src/auth/login.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { LoginReturnDataDTO } from './login-return-data.dto';
+import { ethers } from 'ethers';
+
+@Injectable()
+export class LoginService {
+    constructor(private readonly configService: ConfigService) {}
+
+    public async mbIdentifiers(jwtToken?: string): Promise<Partial<LoginReturnDataDTO>> {
+        const jwtEncodedPayload = jwtToken.split('.')[1];
+        const jwtDecodedPayload = ethers.utils.base64.decode(jwtEncodedPayload);
+        const jwtPayload = JSON.parse(String.fromCharCode.apply(null, jwtDecodedPayload));
+        const userDID = jwtPayload?.iss || 'did:ethr:0x';
```

Comment:
I think we should fail if `iss` is not set, also I don't think passport-did-auth will pass through the jwt without `iss` field.

Reply:
now throws error.
tested with invalid jwt and the error from passport-did-auth was UnhandledPromiseRejection.